templates/blocks/share_buttons.twig

```diff
@@ -10,14 +10,14 @@
                 nativeShareButtons.forEach(nativeShareButton => nativeShareButton.style.display = 'block');
             });
 
-            const dataLayerPush = platform => dataLayer.push({
+            dataLayerPush = platform => dataLayer.push({
```

Comment:
Is that why you removed the `const`? I'm not sure you can do that in JS 🤔

Reply:
Yes, I wasn't sure either.
I have other way -
```
            window.dataLayerPush = window.dataLayerPush || [];
            dataLayerPush = platform => dataLayer.push({
                event: 'uaevent',
                eventCategory: 'Social Share',
                eventAction: platform,
                eventLabel: '{{ social.link }}',
            });

            window.nativeShare = window.nativeShare || [];
            nativeShare = async () => {
                try {
                    await navigator.share({
                        title: '{{ social.title }}',
                        url: '{{ share_url ?? social.link }}',
                        text: '{{ share_text ?? social.description }}',
                    });
                    dataLayerPush('Native');
                } catch (err) {
                    console.log(err);
                }
            };
```
Can we try this option? 🤔

Reply:
I don't think this is gonna work either, because neither `window.dataLayerPush` and `window.nativeShare` will be defined... And you would still need `const` declarations 🤔 Since this problem isn't related to this ticket and to your changes, and the functionality works as expected anyway, maybe we can open a separate ticket to refactor this?

Reply:
https://jira.greenpeace.org/browse/PLANET-7108
Ticket created 👍  Thanks Maud for the help 🙁